parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -157,7 +157,7 @@ $global:AzureCNIConfDir = [Io.path]::Combine("$global:AzureCNIDir", "netconf")
 # $global:NetworkPolicy = "{{GetParameter "networkPolicy"}}" # BUG: unused
 $global:NetworkPlugin = "{{GetParameter "networkPlugin"}}"
 $global:VNetCNIPluginsURL = "{{GetParameter "vnetCniWindowsPluginsURL"}}"
-$global:IsDualStackEnabled = {{if IsIPv6DualStackFeatureEnabled}}$true{{else}}$false{{end}}
+$global:IsDualStackEnabled = {{if or IsIPv6DualStackFeatureEnabled IsDualStack}}$true{{else}}$false{{end}}

```

Comment:
If `IsIPv6DualStackFeatureEnabled` is not used, I suggest you removing this function and code in AgentBaker. We keep this for dualstack but now a new property will be used so we should remove it.